changelog/10235.bugfix.md

```diff
@@ -0,0 +1,3 @@
+Fix ghost slot issues after a restart. Previously events were searched from
```

Comment:
I would use more boring, descriptive language here as "ghost slots" might not mean anything to most users. e.g. "Duplicate slots after restart".